### PR TITLE
Ethernet

### PR DESCRIPTION
**Pull request by Kavikick**

### CODE REVIEW

File: src/Devices/Ethernet_TC.h [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/*
+Based off Arduino's Ethernet library
```
Comment (prestoncarman) [inline src/Devices/Ethernet_TC.h]: Use the following comment style. For at the start of files, classes and functions.

```
/**
 *
 */
```

**Review by jgfoster — CHANGES_REQUESTED** [review]
While we investigate the other test failures, could you fix the formatting issues?

**Review by jgfoster — APPROVED** [review]
LGTM

**Review by prestoncarman — APPROVED** [review]
LGTM

**Comment by prestoncarman** [comment]
@all-contributors please add @Kavikick for code

**Pull request merged**